example.env

```diff
@@ -6,5 +6,6 @@
 # SERVER ENV
 PORT=''
 USER=''
+LOCALHOST=''
```

Comment:
do you still need this file?

Reply:
I created the example.env file for other people to use if they chose to clone my repo!